src/live_restore/live_restore_fs.c

```diff
@@ -519,22 +519,20 @@ __live_restore_remove_extlist_hole(
 
     WT_ASSERT_ALWAYS(session, __wt_rwlock_islocked(session, &lr_fh->ext_lock),
       "Live restore lock not taken when needed");
-    __wt_verbose_debug2(session, WT_VERB_FILEOPS, "REMOVE HOLE %s: %" PRId64 "-%" PRId64,
+    __wt_verbose_debug2(session, WT_VERB_LIVE_RESTORE, "REMOVE HOLE %s: %" PRId64 "-%" PRId64,
```

Comment:
I think READ / WRITE and extent list operations such as this one should go to level 3. They'll be super spammy otherwise.

Reply:
Done. Although the only change I made was renaming the verbose flag.